docs/iris/src/whitepapers/change_management.rst

```diff
@@ -0,0 +1,393 @@
+.. _change_management:
+
+Change Management in Iris from the User's perspective
+*****************************************************
+
+As Iris changes, user code will need revising from time to time to keep it
+working, or to maintain best practice.  At the very least, you are advised to
+review existing code to ensure it functions correctly with new releases.
+
+Here, we define ways to make this as easy as possible.
+
+.. include:: ../userguide/change_management_goals.txt
+
+
+Key principles you can rely on
+==============================
+
+Iris code editions are published as defined version releases, with a given
+major and minor version number in the version name, "major.minor.xxx",
+as explained in the :ref:`releases section <iris_change_releases>` below.
+
+    * Code that currently works should **still work**, and behave exactly the
+      same, in any subsequent sub-release with the same major release number.
+
+    * The only time we will make changes that can break existing code is at
+      a **major release**.
+
+    * At a major release, code that works **and emits no deprecation warnings**
+      in the latest previous (minor) release should still work, and behave
+      exactly the same.
+
+
+**What can possibly go wrong ?**
+
+If your code produces :ref:`deprecation warnings <iris_deprecations>`, then it
+*could* behave differently, or fail, at the next major release.
+
+
+
+User Actions : How you should respond to changes and releases
+=============================================================
+
+Checklist :
+
+* when a new **testing or candidate version** is announced
+    if convenient, test your working legacy code against it and report any problems.
+
+* when a new **minor version is released**
+
+    * review the 'Whatsnew' documentation to see if it introduces any
```

Comment:
No need to have 'Whatsnew' as one word, since it's not a link to anything that might be named that way.